src/view/context/view.cpp

```diff
@@ -24,32 +65,48 @@ void Context::View::updateContextIcon()
 
 void Context::View::resetCurrentlyPlaying() const
 {
-	if (content != nullptr)
+	if (albumShouldBeExpandable && horizContent != nullptr)
+	{
+		horizContent->reset();
+	}
+	else if(!albumShouldBeExpandable && content != nullptr)
 	{
 		content->reset();
 	}
 }
 
 auto Context::View::getCurrentlyPlaying() const -> const lib::spt::track &
 {
-	if (content == nullptr)
+	if (albumShouldBeExpandable && horizContent != nullptr)
+	{
+		return horizContent->getCurrentlyPlaying();
+	}
+	else if(!albumShouldBeExpandable && content != nullptr)
 	{
-		throw std::runtime_error("No content");
+		return content->getCurrentlyPlaying();
 	}
-	return content->getCurrentlyPlaying();
+	throw std::runtime_error("No content");
```

Comment:
This is fine, but as we're in Qt, we can also use `qFatal()`, which more or less does the same thing, just the "Qt-way".